configure.ac:34: error: AM_INIT_AUTOMAKE expanded multiple times
/usr/share/aclocal-1.16/init.m4:29: AM_INIT_AUTOMAKE is expanded from...
configure.ac:19: the top level
/usr/share/aclocal-1.16/init.m4:29: AM_INIT_AUTOMAKE is expanded from...
configure.ac:34: the top level
autom4te: error: /usr/bin/m4 failed with exit status: 1
--- a/configure.ac
+++ b/configure.ac
@@ -31,4 +31,3 @@
 
 AC_CONFIG_FILES([Makefile config.h])
 AC_OUTPUT
-AM_INIT_AUTOMAKE
